lib/rucio/transfertool/fts3.py

```diff
@@ -1369,10 +1370,7 @@ def set_se_config(
 
         params_dict = {storage_element: {'operations': {}, 'se_info': {}}}
         if staging is not None:
-            try:
-                policy = config_get('policy', 'permission')
-            except Exception:
-                self.logger(logging.WARNING, 'Could not get policy from config')
+            policy = get_policy()
```

Comment:
I think this is fine, but worth pointing out that the behaviour here will change when policy is not found:
- beforehand, we'd log a warning and error out on the next line `params_dict[storage_element]['operations'] = {policy: {'staging': staging}}`, because `policy` would be undefined
- now, we fall back to `def` and no warning is logged, and execution continues

Reply:
I've added a warning to `get_policy` as discussed